config/jsonconfig/config.go

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Also put the comments for a function above the function.  This is also pretty standard will help with Go Docs